setup.cfg

```diff
@@ -34,7 +34,7 @@ setup_requires = setuptools_scm
 include_package_data = True
 install_requires =
     numpy>=1.23
-    astropy>=4.3
+    astropy>=6.0
```

Comment:
I am fine with this change, of course, but I will note that `setup.cfg` changes here conflict with those made in PR #1755. I think one of these PRs need to revert the changes to this file.

Reply:
Thanks for pointing this out.  I'm working on consolidating this...